cmd/webhooks/list.go

```diff
@@ -34,17 +35,17 @@ func NewListCommand() *cobra.Command {
 				WithWriter(cmd.OutOrStdout()).
 				WithData(
 					internal.Prepend(
-						internal.Map(response.Cursor.Data, func(src webhookclient.Config) []string {
-							return []string{
-								*src.Id,
-								src.CreatedAt.Format(time.RFC3339),
-								internal.StringPointerToString(src.Secret),
-								*src.Endpoint,
-								internal.BoolPointerToString(src.Active),
-								strings.Join(src.EventTypes, ","),
-							}
-						}),
-						[]string{"ID", "Created at", "Secret", "Endpoint", "Active", "Event types"},
+						internal.Map(res.Cursor.Data,
+							func(src formance.ConfigActivated) []string {
+								return []string{
+									src.CreatedAt.Format(time.RFC3339),
+									internal.StringPointerToString(src.Secret),
+									*src.Endpoint,
+									internal.BoolPointerToString(src.Active),
+									strings.Join(src.EventTypes, ","),
+								}
+							}),
+						[]string{"Created at", "Secret", "Endpoint", "Active", "Event types"},
```

Comment:
Why remove ID?

Reply:
My mistake! I forgot it in the webhooks swagger. I put it back :)